### PR TITLE
Implement add and remove todos

### PR DESCRIPTION
**Pull request by danielochuba**
# To Do list: add & remove
- implementation of add and remove functionalities in the To-do app

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,41 +1,84 @@
...
+  e.preventDefault();
+  const newTask = document.getElementById('taskInput').value;
+  const todos = JSON.parse(localStorage.getItem('todos') || '[]');
+
+  const newTodo = {
+    description: newTask,
+    completed: false,
+    id: Date.now(),
+  };
+
```
Comment (Danieldotcomcoder) [inline src/index.js]: The index property should be set to the value of the new array length as mentioned in the requirments, currently you have set it as a Date. Kindly fix this issue.


image

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @danielochuba , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.js (hunk 1 above)
Comment (danielochuba) [reply src/index.js]: @Danieldotcomcoder 👋 thank you for pointing that out, I have now fixed it

**Review by Lookingfor00 — APPROVED** [review]
# Approved 🟢
Hi @danielochuba , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
allfather-approves-approved

## Highlights
- Project look awsome 🍏 🌷 
- Code is clean and well organized 🧼 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@Trast00) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
Hey @danielochuba,



According to this review [summary](https://github.com/danielochuba/TO-DO-LIST-M_R/pull/2#pullrequestreview-1415576138). Your project has been approved:

Cheers, and Happy Coding!

**Pull request merged**